src/shapes/FocusedInputShape.js

```diff
@@ -5,4 +5,10 @@ import {
   END_DATE,
 } from '../../constants';
 
-export default PropTypes.oneOf([START_DATE, END_DATE]);
+export default function(props, propName, componentName){
+  const item = props[propName];
+  if (item === START_DATE || item === END_DATE || item === ''){
+    return null;
+  }
+  return new Error(`${propName} must be an empty string, "${START_DATE}", or "${END_DATE}"`);
```

Comment:
`PropTypes.oneOf` handles this already. if you want to allow an empty string, do `PropTypes.oneOf([START_DATE, END_DATE, ''])` - but I'm not sure why we'd want to allow an empty string.